src/command/dev/next/router/binary.rs

```diff
@@ -0,0 +1,17 @@
+use camino::Utf8PathBuf;
+use semver::Version;
+
+#[derive(Clone, Debug)]
+#[cfg_attr(test, derive(derive_getters::Getters))]
+pub struct RouterBinary {
+    #[allow(unused)]
+    exe: Utf8PathBuf,
+    #[allow(unused)]
+    version: Version,
+}
+
+impl RouterBinary {
+    pub fn new(exe: Utf8PathBuf, version: Version) -> RouterBinary {
+        RouterBinary { exe, version }
+    }
+}
```

Comment:
would we want to use Builder here instead? it seems like an emerging convention

Reply:
My litmus test has been that more than two arguments equates to wanting a builder. Otherwise, a constructor is usually good enough.